google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/EnhancedBigtableStub.java

```diff
@@ -327,6 +327,36 @@ public EnhancedBigtableStub(EnhancedBigtableStubSettings settings, ClientContext
     return traced.withDefaultCallContext(clientContext.getDefaultCallContext());
   }
 
+  /**
+   * Creates a callable chain to handle point ReadRows RPCs. The chain will:
+   *
+   * <ul>
+   *   <li>Convert a {@link Query} into a {@link com.google.bigtable.v2.ReadRowsRequest} and
+   *       dispatch the RPC.
+   *   <li>Upon receiving the response stream, it will merge the {@link
+   *       com.google.bigtable.v2.ReadRowsResponse.CellChunk}s in logical rows. The actual row
+   *       implementation can be configured in by the {@code rowAdapter} parameter.
+   *   <li>Retry/resume on failure.
+   *   <li>Filter out marker rows.
+   * </ul>
+   *
+   * <p>NOTE: the caller is responsible for adding tracing & metrics.
+   */
+  protected <RowT> UnaryCallable<Query, RowT> createReadRowRawCallable(
```

Comment:
I wonder if we could just reuse `#createReadRowsRawCallable`: https://github.com/googleapis/java-bigtable/blob/main/google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/EnhancedBigtableStub.java#L294

So in `BigtableChannelPrimer` we could do something like:
```
ApiFuture<Row> f =
     stub.createReadRowsRawCallable(new DefaultRowAdapter())
        .first()
        .futureCall(
            ReadRowsRequest
                 .newBuilder()
                 .setTableName(TableName.format(primingSettings.getProjectId(), primingSettings.getInstanceId(), tableId))
                       ...)
```

Reply:
Oh that is a great way to reduce adding more callables to the Stub, done!